docker/README.md

```diff
@@ -1,18 +1,28 @@
 # Get docker images
 
-You can either build the images locally, or get our "latest stable" version from the Docker hub. We recommend using the Docker hub option.
+You can either build the images locally, or get the images from the Docker hub. We recommend using the Docker hub option.
 
 ## Download images from the Docker hub
 
+**openSUSE images**
```

Comment:
Any reason why not to make it a 3rd level header? Later in the docs there is a 3rd level header like this just for one sentence, IMO I would appreciate a link from index directly to download commands.